doofinder.php

```diff
@@ -35,15 +35,15 @@ class Doofinder extends Module
     const DOOMANAGER_URL = 'https://admin.doofinder.com';
     const GS_SHORT_DESCRIPTION = 1;
     const GS_LONG_DESCRIPTION = 2;
-    const VERSION = '4.6.5';
+    const VERSION = '4.7.0';
     const YES = 1;
     const NO = 0;
 
     public function __construct()
     {
         $this->name = 'doofinder';
         $this->tab = 'search_filter';
-        $this->version = '4.6.5';
+        $this->version = '4.7.0';
```

Comment:
```suggestion
        $this->version = '4.6.6';
```

Reply:
Ok, I misunderstood this part, changed to patch version: https://github.com/doofinder/doofinder-prestashop/pull/143/commits/13a4e4631a3fff59967320600ca685099550cd59